ajax/libs/remark/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "remark",
+  "version": "0.14.1",
+  "dependencies": {
+    "marked": "0.3.1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/gnab/remark.git"
+  },
+  "license": "MIT",
+  "description": "A simple, in-browser, markdown-driven slideshow tool",
+  "keywords": [
+    "slides",
+    "slideshow",
+    "markdown",
+    "templates"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/gnab/remark.git",
+    "fileMap": [
+      {
+        "basePath": "out",
```

Comment:
I'm not sure exactly how the auto-update script works, but this would look for this directory in the tag tree? e.g. `https://github.com/gnab/remark/tree/v0.14.0`